train-kubernetes.gemspec

```diff
@@ -45,6 +45,6 @@ Gem::Specification.new do |spec|
 
   # All plugins should mention train, > 1.4
   # pinning k8s-ruby to 0.17.0 since it has support for Ruby version 3.4
```

Comment:
can we update/remove the comment as well, since we are changing the pin for k8s-ruby